docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,50 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 12 April 2023
+
+Frontend version: 1.0.0-20230406.0949 | Backend version: 1.58.2-230406.0515
+
+**Feature** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Search bar added on TechPass portal</summary>
+
+We have updated TechPass portal with a new search bar feature that allows users to search for specific applications. Additionally, tenants can also use the search bar to search for applications and OTP Credentials.
+
+</details>
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Updated tenants' restriction on TechBiz application role assignment & groups management in TechPass </summary>
+
+Previously, tenants were not allowed to manage their applications' role assignments on TechPass if they had onboarded to TechBiz. However, we understand that some applications may not be on TechBiz, so we have updated the restriction to be applied at the application-level instead. 
+
+Additionally, TechBiz-created groups will no longer be displayed on the TechPass portal, and tenants will not be able to call certain Automation APIs for these groups. To help tenants exclude TechBiz-created groups from their results, we have introduced a new *excludeOnTechBiz* query parameter for the **List Groups** Automation API. These changes are designed to streamline the management of application role assignments and groups for TechPass tenants.
```

Comment:
What is the default value of _excludeOnTechBiz_ ?

Reply:
default is `false` currently , will change to `true`

Reply:
actually would it be better to change the param to `includeOnTechBiz` & have it `false` by default?

Reply:
ok let me what i need to change to

Reply:
no need to change the release notes, i'll change the implementation in the next release

Reply:
yes, agree with includeOnTechBiz and set the default to false

Reply:
this way the portal and API have consistent default behaviour